units/host_status.go

```diff
@@ -185,8 +184,8 @@ func (j *cloudHostReadyJob) setCloudHostStatus(ctx context.Context, m cloud.Mana
 
 		catcher := grip.NewBasicCatcher()
 		if h.SpawnOptions.SpawnedByTask {
```

Comment:
It seems like checking if it's spawned by a task is redundant if it's already checked in the function.

Reply:
removed the extra check